databricks/koalas/tests/test_dataframe_spark_io.py

```diff
@@ -89,8 +89,8 @@ def check(columns, expected):
             expected_idx = expected.set_index("bhello")[["f", "i32", "i64"]]
             actual_idx = ks.read_parquet(tmp, index_col="bhello")[["f", "i32", "i64"]]
             self.assert_eq(
-                actual_idx.sort_values(by="f").to_spark().toPandas(),
-                expected_idx.sort_values(by="f").to_spark().toPandas(),
+                actual_idx.sort_values(by="f").to_spark().to_pandas(),
+                expected_idx.sort_values(by="f").to_spark().to_pandas(),
```

Comment:
We can't change these since Spark DataFrame doesn't have `to_pandas()`.

Reply:
ditto for the following changes.

Reply:
Why are we using `to_spark()` here, btw ..? @HyukjinKwon do you remember the reason?

Reply:
Oh, sorry my bad. I changed whole `toPandas()` to `to_pandas()` at once with grep. 😅

Reply:
I don't remember .. I think we can just remove `to_spark` call.

Reply:
It's for dropping the `index_col` since the order of indices could be different.